### ISSUE TITLE
brew install beta/latest version.

### ISSUE THREAD

**Issue by Kamik423** [MacDownApp/macdown#899]
When installing with brew I can only install `0.7.1`. As far as I can tell there is no way to update to `0.7.2` with brew alone. Updating from within MacDown yields a 
```
An error occurred while extracting the archive. Please try again later.
```

**Comment by franklinyu** [comment]
1. [Homebrew Cask](https://caskroom.github.io/) does not support flags, so we cannot do `--beta` or `--testing`. It was estimated that few people using testing branch would like to manage that through package manager. We can make another Cask Formula if desired.
2. The error message is unexpected. It would be appreciated if a screenshot is available (and I'll try it myself tonight).

**Comment by Kamik423** [comment]
1) [this](https://github.com/caskroom/homebrew-versions) seems to support beta versions
2) open a new Issue: #900

**Comment by Kamik423** [comment]
Yes. Close it if you feel like having beta Formula is not necessary.

**Comment by franklinyu** [comment]
Well, at least a "nice to have".
